### PR TITLE
Stocks app

### PR DESCRIPTION
**Pull request by matovu-farid**
In this branch, the following changes were added:
- The home page was added
- The details page was added
- Integration Tests were added to test interactions
- A pure function test was added to test the search logic
- The homepage data was fetched from the financial API and added
- The functionality onClick of each company was added in order to navigate to the details page
- The data for the details page was fetched from the financial API and displayed
- The mobile app was styled
- The app was deployed and added the new link to the README file
- A video was recorded and the link added to the README file
- The README file was updated with the app description
- Original design was attributed to the author of the design

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,46 @@
...
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+👤 **Matovu Farid Nkoba**
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+- GitHub: [@matovu-farid](https://github.com/matovu-farid)
+- Twitter: [@matovu100](https://twitter.com/matovu100)
+- LinkedIn: [matovu-farid](https://www.linkedin.com/in/matovu-farid-48b80257)
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+Original design idea by [Nelson Sakwa on Behance.](https://www.behance.net/sakwadesignstudio)
```
Comment (mikethreels) [inline README.md]: - [ ] according to the requirements The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). please have a look at [the readme templat](https://github.com/microverseinc/readme-template)e provided by Microverse for an example.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Farid, 

Great job on the project so far, the design is great and it works mostly as expected 👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] according to the requirements the video should contain a highlight about a piece of the code you finds interesting or you are proud of making, please record the video again and include this part as well
- [ ] The first link 'S&P 500' is not working

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by matovu-farid** [comment]
The Snp 500 is not meant to be a link, it's meant to be a general category

**Review by mmsesay — APPROVED** [review]
Hi @matovu-farid,

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**